addons/mail/static/src/components/composer_suggested_recipient/composer_suggested_recipient.js

```diff
@@ -84,16 +116,27 @@ class ComposerSuggestedRecipient extends Component {
      * @private
      */
     _onChangeCheckbox() {
+        const isChecked = this._checkboxRef.el.checked;
+        this.suggestedRecipientInfo.update({ isSelected: isChecked });
         if (!this.suggestedRecipientInfo.partner) {
-            // Disable the checkbox to avoid multiple click otherwise multiple
-            // bugged modal will be open
-            this.checkboxRef.el.disabled = true;
```

Comment:
If a user does multiple clicks on the checkbox (like a double/triple click), it will open multiple modals.

Reply:
should be solved